cmd/oras/pull.go

```diff
@@ -83,6 +85,8 @@ Example - Pull files with local cache:
 }
 
 func runPull(opts pullOptions) error {
+	set := make(map[string]void)
```

Comment:
bool defaults to false
```suggestion
	printed := make(map[string]bool)
```